CMakeLists.txt

```diff
@@ -104,10 +106,18 @@ if(${VELOX_ENABLE_EXAMPLES})
   set(VELOX_ENABLE_TEST_UTILS ON)
 endif()
 
-if(${VELOX_ENABLE_BENCHMARKS} OR ${VELOX_ENABLE_BENCHMARKS_BASIC})
```

Comment:
Should we just get rid of VELOX_ENABLE_BENCHMARKS* ? And just have VELOX_BUILD_BENCHMARKS, VELOX_BUILD_BENCHMARKS_BASIC ?

Reply:
Maybe that's part of a broader discussion, but I think we need to have a differentiation between external flags that users can toggle on/off (the *_BUILD_* ones), and internal flags we use to decide which modules to compile (the *_ENABLE_* flags).

For instance, a user would think it's perfectly reasonable to do something like:

-DVELOX_BUILD_MINIMAL=ON -DVELOX_ENABLE_AGGREGATES=ON

but that doesn't do what you would expect (it doesn't compile aggregates) because of how cmake handles these flags internally. Not sure how Arrow solves this issue though.